Framework/Kernel/test/TimeSeriesPropertyTest.h

```diff
@@ -1345,6 +1345,47 @@ class TimeSeriesPropertyTest : public CxxTest::TestSuite {
     return;
   }
 
+  /*
+   * Test cloneWithTimeShift
+   */
+  void test_cloneWithTimeShift() {
+    TimeSeriesProperty<int> *time_series =
+        new TimeSeriesProperty<int>("IntUnixTest");
+    time_series->addValue("2019-02-10T16:17:00", 1);
+
+    TimeSeriesProperty<int> *time_series_small_shift =
```

Comment:
These _could_ be changed to unique_ptr, but the init is kind of ugly:
`std::unique_ptr<TimeSeriesProperty<int>>(<TimeSeriesProperty<int> *>(time_series->cloneWithTimeShift(100.)));`.

Up to you if you want to change them or leave them. I don't mind either way

Reply:
I've decided to leave them as it's slightly cleaner